far/viewer.cpp

```diff
@@ -1007,11 +1011,20 @@ void Viewer::SetStatusMode(int Mode)
 	ShowStatusLine=Mode;
 }
 
+static constexpr std::array extra_word_divs{ encoding::bom_char, encoding::replace_char };
```

Comment:
Come to think of it, should we consider replace_char non-word at all?
The domain of word characters is way larger than the domain of punctuation characters, so any encoding errors are likely to be within words.

Reply:
Even though an arbitrary corrupted byte is more likely to fall on a word character, it feels that a false positive (found when should not) is more appropriate than false negative (not found when should) when interactively presenting results to a human being.

That said, I'd rather do not change the logic here in this PR. The risk of breaking something is high enough as it is. Let's make this change in a separate commit to facilitate rollback if necessary.

Reply:
Let's leave it as is for now.